Automattic/simplenote-ios
# General Colour Adjustments

### Fix
iOS dark colours have been pulled directly from Android. This is a PR to replace the Android palette with an iOS standard dark theme as well as update some colour discrepancies.

Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/36532468/74775443-a62c2a00-528d-11ea-8478-95e4b2ed7ef1.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74775220-2aca7880-528d-11ea-9d68-e59ce76f2ddc.PNG)
![](https://user-images.githubusercontent.com/36532468/74775464-b2b08280-528d-11ea-8b29-c0fab6a5ccc6.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74775249-37e76780-528d-11ea-8229-a134ebac4e6e.PNG)
![](https://user-images.githubusercontent.com/36532468/74775497-c1973500-528d-11ea-9860-00bfd8793bfc.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74775265-43d32980-528d-11ea-89ec-1e07d6cdf0c2.PNG)
![](https://user-images.githubusercontent.com/36532468/74775531-cbb93380-528d-11ea-971b-3abbb1578a92.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74775294-50f01880-528d-11ea-8f48-9a6628642fa9.PNG)
![](https://user-images.githubusercontent.com/36532468/74761987-e92dd380-5274-11ea-94cf-091eaaccdfa4.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74761729-7290d600-5274-11ea-9e39-800457571e88.PNG)
![](https://user-images.githubusercontent.com/36532468/74762031-f480ff00-5274-11ea-83f6-45a313000a40.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74761752-7f152e80-5274-11ea-8943-aeec905813d8.PNG)
![](https://user-images.githubusercontent.com/36532468/74762061-02cf1b00-5275-11ea-901f-8a446e8921a3.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74761774-8a685a00-5274-11ea-8a15-5198f5537833.PNG)
![](https://user-images.githubusercontent.com/36532468/74762161-2c884200-5275-11ea-87f2-bde4bf493364.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74761811-9a803980-5274-11ea-99b4-00ae8a52da2d.PNG)

### Test
1. Switch app to light theme
2. Notice light theme has been updated to iOS standards
3. Switch app to dark theme
4. Notice dark theme has been updated to iOS standards

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
